DESCRIPTION

```diff
@@ -28,5 +28,6 @@ Imports:
     e1071,
     randomForest
 Suggests: 
-    testthat
+    testthat, knitr, rmarkdown
```

Comment:
Do `Suggests` packages need to be separated into a new line like `Imports`?

Reply:
They work well this way.